truffle.js

```diff
@@ -0,0 +1,2 @@
+module.exports = {
+};
```

Comment:
why do we need this at the top-level of the monorepo?

Reply:
truffle needs to have this to be able to start the private blockchain. I was thinking that it would be nice to be able to start the private blockchain from the root of the hub. truffle wants to see this in your working directory when you start the private blockchain, otherwise it will complain and just not start.

Reply:
(this is where you put all your truffle config, but since the defaults are really nice, we have no truffle config)...